app/views/templates/script/package_action.erb

```diff
@@ -90,6 +95,10 @@ handle_zypp_res_codes () {
   [ $RETVAL -eq 0 ] || exit_with_message "Pre script failed" $RETVAL
 <% end -%>
 
+<% if !input("refresh_subscription").blank? && input("refresh_subscription") == "true" -%>
```

Comment:
yes, this works, too. Actually, personally I prefer to have it crystal clear written instead of "hoping" that the second part always works. But, it works :-)